Essentials/src/com/earth2me/essentials/commands/Commandmute.java

```diff
@@ -66,17 +79,41 @@ public void run(final Server server, final CommandSource sender, final String co
 
             if (muted) {
                 if (muteTimestamp > 0) {
-                    sender.sendMessage(tl("mutedPlayerFor", user.getDisplayName(), muteTime));
-                    user.sendMessage(tl("playerMutedFor", muteTime));
+                    if (user.getMuteReason ().equals ("")) {
+                        sender.sendMessage(tl("mutedPlayerFor", user.getDisplayName(), muteTime));
+                        user.sendMessage(tl("playerMutedFor", muteTime));
+                    }
+                    else {
```

Comment:
Another style issue - `else`s should be on the same line as the previous block's closing bracket.